easybuild/easyconfigs/d/detettore/detettore-2.0.2-foss-2021a-Python-3.9.5.eb

```diff
@@ -0,0 +1,42 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Iñaki Mtz de Ilarduya Muñoz
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'PythonPackage'
+
+name = 'detettore'
+version = '2.0.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/cstritt/detettore'
+description = """A program to detect transposable element polymorphisms"""
+
+source_urls = ['https://pypi.python.org/packages/source/d/detettore']
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['7f290c99f65a49a27d6f2468b05706ea9191af105e758875c22e432533a94218']
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('minimap2', '2.20'),
+    ('Pysam', '0.18.0', '-Python-%(pyver)s'),
```

Comment:
We don't have an easyconfig file for Pysam 0.18.0 yet, and if we would, it (also) wouldn't have a versionsuffix; that's also why the tests are failing currently.

We're already using Pysam 0.16.0.1 as dependency in various other easyconfigs using the `foss/2021a` toolchain, so it would be better to also stick to that here (so the `detettore` can be loaded together with other modules using the `foss/2021a` toolchain without running into version conflicts).

Reply:
@boegel I have refactored the easyconfig based on the feedback from the CI and the output from `sanity_pip_check = True` which I had omitted. 

I agree about the Pysam dependency, but it seems fixed at `0.17.0` on `setup.py`.  I have asked the author of the program if using `0.16.0.1` would be an option and will update the config if we can use the same Pysam as the one default to the toolchain.